### DISCUSSION TITLE
German Vits model finetuning -> output has increased speed and way higher pitch

### DISCUSSION THREAD

**Discussion by buczekEngineering** [coqui-ai/TTS#1643] [General Q&A] [answered]
I fine tuned the @thorstenMueller German Vits model, recently released. I used 1 hours of 22050hz sampled 16bits pcm, mono audio, female German voice. 
I used exact the same configuration (adjusted only the paths to datasets, phoneme cache etc). The model is fine tuning fast and the output already after 9000 steps is quite good. However the  synthesized speech has much higher pitch then the original speech from my training data.
Does anybody know why?  Did i miss some parameter?

**Comment by domcross** [comment]
Maybe your audio files have wrong header information and/or are encoded in a wrong way.
When playing the audio files in VLC (or another media player) do they sound correct?
Can you check the header information e.g. with the `soxi` command from [SOX](http://sox.sourceforge.net)?

**Comment by buczekEngineering** [comment]
Input File     : 'cf8c075c309055e83f7b3632cb0fda46.wav'
Channels       : 1
**Sample Rate    : 16000**
Precision      : 16-bit
Duration       : 00:00:01.78 = 28528 samples ~ 133.725 CDDA sectors
File Size      : 57.1k
Bit Rate       : 256k
Sample Encoding: 16-bit Signed Integer PCM

My bad, the sample rate was indeed 16kHz, but dunno if this could at any point impact the pitch(?), I would rather say not. Correct me please if I'm wrong.The audio sounds correct once played in the VLC.

**Comment by domcross** [comment, marked as answer]
Your 16K-sample is interpreted as 22K, this is a speed increase of approx 1.37x, the pitch changes by roughly 3.5 semitones - this is notably

**Comment by buczekEngineering** [comment]
I forgot to say thank you to @domcross  and @thorstenMueller for preparing this pre-trained model. The quality of the pre-trained model is really amazing. Good job guys :1st_place_medal: 

The suggested by @domcross solution solved the problem with my fine tuning. The fine-tuned model is also very good and fast now (RTF:0.3).

**Reply by Rico2000** [reply]
@buczekEngineering 
Can you send a link for the  pre-trained model?
Would be great could not find it 
Thanks

**Reply by domcross** [reply]
It is part of the v0.7.0 models release: https://github.com/coqui-ai/TTS/releases/tag/v0.7.0_models

**Reply by yurtmete** [reply]
@buczekEngineering I do have some questions about the fine-tuning, 

1- Does the pre-trained model come **with** the discriminator?
2- How did you adjust the learning rates when you started the fine-tuning? 
3- Did you use only your female speaker's samples during the fine-tuning, or did you also include pre-trained model speaker's voice to the training set?

Thank you in advance.
